discordbio/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "MauriceX",
+    "id": "268768664411439115"
+  },
+  "service": "Discord.bio",
+  "description": {
+    "en": "Create a profile in under 30 seconds.",
+    "de": "Erstelle ein Profil in weniger als 30 sekunden."
+  },
+  "url": ['discord.bio'],
```

Comment:
"url": "discord.bio",

Reply:
ohh thanks - i thought the [] is important

Reply:
[] is for arrays